packages/medusa/src/api/routes/admin/price-lists/create-price-list.ts

```diff
@@ -39,6 +42,9 @@ import { EntityManager } from "typeorm"
  *             enum:
  *              - sale
  *              - override
+ *        includes_tax:
+ *             description: "[EXPERIMENTAL] Is the price list tax inclusive or not"
```

Comment:
suggestion: Rephrase to improve semantics
```suggestion
 *             description: "[EXPERIMENTAL] Tax included in prices of price list"
```